server/sonar-web/src/main/js/components/ui/Avatar.js

```diff
@@ -29,6 +29,7 @@ export default class Avatar extends React.Component {
   };
 
   render () {
+    // FIXME
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1134)